src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparatorCommentPackage.java

```diff
@@ -0,0 +1,6 @@
+//non-compiled with javac: non-compilable annotation, for testing
+@Deprecated
+package com.puppycrawl.tools.checkstyle.checks.whitespace.emptylineseparator;
```

Comment:
Please explain what is a problem here

Reply:
This was to kill this mutation: http://rveach.no-ip.org/checkstyle/regression/pitest-reports/30/pitest-whitespace/com.puppycrawl.tools.checkstyle.checks.whitespace/EmptyLineSeparatorCheck.java.html#org.pitest.mutationtest.report.html.SourceFile@7d2fc711_422

Breif: To meet the conditions, we need a package with a annotation that is stored in a file that isn't package-info.java .
Specifics: The rest of the conditions in the method have to be met except for the line that is the surviving mutant. Child count of 0 is because there is only 1 annotation, which is the first child of the package definition, and it has no empty line or javadoc before it. Package must also be on line 2+. The mutation's condition has to be it's inverse to satisfy killing the mutation, so it must be a file that isn't `package-info.java`. This is what came up when I did a regression search as we have a similar file checkstyle elsewhere.

File isn't compilable because only package-infos can have annotations and we need this to be a non package-info file.

Reply:
>File isn't compilable because only package-infos can have annotations and we need this to be a non package-info file.

By this we damage whole idea keeping inputs compilable/real. We tried our best to avoid running on files that are not passing compilation.

I am not ready to provide  solution for this now, but probably code need to be rewritten.
Let's keep this case in ignore list of survived and do not create this inpuy

Reply:
Done.

Reply:
Suppression should have comment "till " , and in issue all your details from above.

Reply:
If we find no way to resolve it, we will create probably new group in uncompiled - "for mutation testing"

Reply:
Done.